packages/cactus-plugin-htlc-eth-besu-erc20/src/main/solidity/token-erc20-contract/DemoHelpers.json

```diff
@@ -0,0 +1,6078 @@
+{
```

Comment:
@petermetz They were copied from test package because I need to test openapi from this package to be able to access openapi.json for send it to function installOpenapiValidationMiddleware. I just changed directory to test folder.

Reply:
@elenaizaguirre Can't you get a hold of the open API spec contents the same way the api-server.ts file does? (which would then eliminate the need to duplicate these files)

Reply:
@petermetz Done!